examples/workflows/experimental/new_container_decorator.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional
+
+from typing_extensions import Annotated
+
+from hera.shared import global_config
+from hera.workflows import Container, Input, Output, Parameter, WorkflowTemplate
+
+global_config.experimental_features["script_annotations"] = True
+global_config.experimental_features["script_pydantic_io"] = True
```

Comment:
Can we consolidate all of the flags?

Reply:
Yeah, was thinking to do a small PR on top as it'll probably touch other parts of the codebase

Reply:
Small one here https://github.com/argoproj-labs/hera/pull/1076